src/main/kotlin/net/ccbluex/liquidbounce/utils/aiming/anglesmooth/AccelerationSmoothMode.kt

```diff
@@ -17,15 +17,18 @@ import kotlin.math.roundToInt
 
 class AccelerationSmoothMode(override val parent: ChoiceConfigurable<*>) : AngleSmoothMode("Acceleration") {
 
-    private val yawAcceleration by floatRange("YawAcceleration", 20f..25f, 1f..100f)
-    private val pitchAcceleration by floatRange("PitchAcceleration", 20f..25f, 1f..100f)
+    private val yawAcceleration by floatRange("YawAcceleration", 20f..25f, 1f..180f)
+    private val pitchAcceleration by floatRange("PitchAcceleration", 20f..25f, 1f..180f)
     // TODO: figure out how to implement lower accel bound
     //private val minAcceleration by float("MinAcceleration", -25f, -100f..0f)
     private val yawAccelerationError by float("YawAccelerationError", 0.1f, 0f..1f)
     private val pitchAccelerationError by float("PitchAccelerationError", 0.1f, 0f..1f)
     private val yawConstantError by float("YawConstantError", 0.1f, 0f..10f)
     private val pitchConstantError by float("PitchConstantError", 0.1f, 0f..10f)
 
+    private val yawAccel = yawAcceleration.random().toFloat()
+    private val pitchAccel = pitchAcceleration.random().toFloat()
```

Comment:
What? Now it will not randomize at all.

Reply:
so thats why this shit was rotating slow...